### DISCUSSION TITLE
I am getting 404: Not found error on deploying MERN stack app

### DISCUSSION THREAD

**Discussion by atharvabhuse** [vercel/community#1691] [Help] [answered]
### Summary

I deployed MERN stack app via github to vercel. I encountered status: 404, CODE_NOT_DOUND error.
please, can someone tell me what to do now?

![Screenshot (195)](https://user-images.githubusercontent.com/99831910/222959561-2e9086f5-d87a-4bb8-9710-fec425a48e7a.png)

### Additional information

https://ecommerce-atharva.vercel.app/

### Example

https://github.com/atharvabhuse/ecommerce

**Comment by belauzas** [comment]
I had the same issue. No solution yet.

**Comment by amyegan** [comment, marked as answer]
@atharvabhuse @belauzas If you're having trouble with the backend deployment, I recommend following the guide on [using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel). In my Express example apps, I put my Express functions in a root level `/api` directory and use [rewrites](https://vercel.com/docs/concepts/projects/project-configuration#rewrites) to feed URL paths to the Express function. That strategy has been reliably successful in my experience.

If you have tried that and are still having trouble, please share more with us about how you're deploying your MERN app 🙏
